src/formio/components/jest-util.js

```diff
@@ -0,0 +1,5 @@
+const getComponent = element => element.closest('[ref="component"]');
```

Comment:
I think this will be mega-confusing with getting a formio component instance from the `WebForm`. Maybe rename it?

```suggestion
const getComponentNode = element => element.closest('[ref="component"]');
```

Reply:
Mhh yeay, you might be right.. getComponentNode sounds a lot better.. 👍